recipes/VideoBots.py

```
@@ -8,6 +8,7 @@
 from django.db.models import Q, QuerySet
 from furl import furl
 from pydantic import BaseModel, Field
+from loguru import logger
```

Ruff (0.12.2)

`loguru.logger` imported but unused

Remove unused import: `loguru.logger`

(F401)